### ISSUE TITLE
Will be there a chance that PqDL works again?

### ISSUE THREAD

**Issue by dsiggi** [leoluk/PqDL#2]
Will be there a chance thar PqDL works again?

```
python2.7 PQdl.py -u xyz -p xyz --nobrowser --allsecure --noupdate
INFO - root -> PQdl v0.3.5 (stable) by leoluk. Updates and help on www.leoluk.de/paperless-caching/pqdl
INFO - main -> Update check skipped. Please check for updates yourself!
INFO - main.login -> Logging in as xyz
Traceback (most recent call last):
  File "PQdl.py", line 1006, in <module>
    main()
  File "PQdl.py", line 767, in main
    else ""))
  File "PQdl.py", line 573, in login_gc
    self.select_form(name="aspnetForm")
  File "/usr/lib64/python2.7/site-packages/mechanize/_mechanize.py", line 524, in select_form
    raise FormNotFoundError("no form matching "+description)
mechanize._mechanize.FormNotFoundError: no form matching name 'aspnetForm'
```

**Comment by crdx** [comment]
Take a look at #3

**Comment by leoluk** [comment]
Closed via #3, thank you @crdx
